README.md

```diff
@@ -1,2 +1,184 @@
 # trampss-redux-data-store
+
+Factory of Redux reducers and their associated actions and selectors.
+> Make your Redux code base tinier and simpler to maintain
+
 [![CircleCI](https://circleci.com/gh/Trampss/trampss-redux-data-store.svg?style=shield)](https://circleci.com/gh/Trampss/trampss-redux-data-store) [![Coverage Status](https://coveralls.io/repos/github/Trampss/trampss-redux-data-store/badge.svg?branch=master)](https://coveralls.io/github/Trampss/trampss-redux-data-store?branch=master) [![NPM Version](https://badge.fury.io/js/trampss-redux-data-store.svg)](https://www.npmjs.com/package/trampss-redux-data-store)
+
+## Contents
+ - [Purpose](#purpose)
+ - [Why](#why)
+ - [Installation](#installation)
+ - [API](#api)
+
+## Purpose
+`trampss-redux-data-store` generates generic reducers, actions and selectors in two lines.
+
+```es6
+import factory from 'trampss-redux-data-store'
+export default factory('id')('api')('todos')
+```
+That's it, you exported a function wich is a reducer to include in a Redux `combineReducer`.
+The exported function also have actions and selectors ! Easy.
+
+In this example, we have a `todos` reducer, it has to be combined into `state.api.todos`
+
+## Why
+Using `redux-saga` make our Redux code base simpler : it's like a key/value store. But one drawback is the amount of duplicate code, each ressource has its reducers, actions and selectors.
+
+To avoid growing Redux code base, and to avoid inconsistency, and pain maintenability, we created this lightweight library (<4Kb) that is a factory of reducers, actions and selectors.
```

Comment:
To avoid Redux code base from growing, inconsistency and lowering maintainability ...